akka-projection-grpc/src/main/scala/akka/projection/grpc/internal/AkkaProjectionGrpcSerialization.scala

```diff
@@ -31,6 +31,16 @@ import scalapb.GeneratedMessage
 import scalapb.GeneratedMessageCompanion
 import scalapb.options.Scalapb
 
+/**
+ * INTERNAL API
+ */
+@InternalApi
+private[akka] trait AkkaProjectionGrpcSerialization {
```

Comment:
Do we have to include AkkaProjection in the name?

Reply:
Dropped